snapcraft/tests/commands/test_update.py

```diff
@@ -20,19 +20,44 @@
 import yaml
 from testtools.matchers import Contains, Equals, FileExists
 from xdg import BaseDirectory
+from textwrap import dedent
+from unittest import mock
+from unittest.mock import call
 
 from snapcraft.tests import TestWithFakeRemoteParts
+from snapcraft.tests import fixture_setup
 from . import CommandBaseTestCase
 
 
 class UpdateCommandTestCase(CommandBaseTestCase, TestWithFakeRemoteParts):
 
+    yaml_template = dedent("""\
+        name: snap-test
+        version: 1.0
+        summary: test snapping
+        description: if snap is succesful a snap package will be available
+        architectures: ['amd64']
+        type: {snap_type}
+        confinement: strict
+        grade: stable
+
+        parts:
+            part1:
+                plugin: nil
+        """)
+
     def setUp(self):
         super().setUp()
         self.parts_dir = os.path.join(BaseDirectory.xdg_data_home, 'snapcraft')
         self.parts_yaml = os.path.join(self.parts_dir, 'parts.yaml')
         self.headers_yaml = os.path.join(self.parts_dir, 'headers.yaml')
 
+    def make_snapcraft_yaml(self, n=1, snap_type='app', snapcraft_yaml=None):
```

Comment:
There is a `make_snapcraft_yaml` in the parent class for tests which you can just use, right?

Reply:
I basically went with the style used in other tests there, which have their own customized version of the method for repeated use. But you're right, in this case I can keep it really simple. So, done.